src/memento_test.jl

```diff
@@ -46,9 +69,10 @@ end
 """
     @test_nolog(logger, level, msg, expr)
 
-Adds a temporary test handler to the `logger` that checks that there was no log record with
-the expected `level` and `msg` before executing the `expr`. The handler is always removed
-after executing `expr`.
+Test that the expression `expr` does not emit a record in the `logger` with the specified
+`level` string and contains the `msg` string or matches the `msg` regular expression.
```

Comment:
Needs rewording for the negative case

Reply:
Ya, I rushed this part. Thanks for calling me out